js/main.js

```diff
@@ -99,60 +74,67 @@ $(document).ready(function() {
 	 * @see: /templates/frontend/pages/userRegister.tpl
 	 */
 	function reviewerInterestsToggle() {
-		var is_checked = isReviewerSelected();
-		if (is_checked) {
-			$('#reviewerInterests').removeClass('hidden');
+		if (checkboxReviewerInterests.checked) {
+			document.getElementById('reviewerInterests').classList.remove('hidden');
 		} else {
-			$('#reviewerInterests').addClass('hidden');
+			document.getElementById('reviewerInterests').classList.add('hidden');
```

Comment:
You might be a little bit careful with this and similar calls. If no element with id `reviewerInterests` exists, calling `classList` will throw an error and probably stop execution of the JS file from this point on.

Reply:
Yeah, had such problems during tests. There is a variant of a null safe operator in JS but its usage increases requirements for browsers, so I've ended up just checking the page-related specifiers  where it's possible not to identify each element's existence turning the code into the deeply nested if-else blocks :)